app/src/main/res/values/strings.xml

```diff
@@ -390,8 +390,10 @@
     <string name="select_region_qr">Select region with QR code</string>
     <!--Customer care details-->
     <string name="need_help">Contact Us</string>
-    <string name="help_line_number" translatable="false">8000000000</string>
-    <string name="contact_email" translatable="false">support@mifos.org</string>
+    <string name="help_line_number"
```

Comment:
These changes are not related to this PR , perhaps you can make a separate PR in which you can address these formatting issues